Sources/Brave/Frontend/Shields/SubmitReportView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  SwiftUIView.swift
+//  
+//
+//  Created by Jacob on 2023-09-21.
+//
+
+import SwiftUI
+import Strings
+import BraveUI
+import DesignSystem
+import BraveShields
+
+struct SubmitReportView: View {
+  let width: CGFloat
+  let url: URL
+  let dismiss: () -> Void
+  let submit: (String, String) -> Void
+  
+  @State var additionalDetails = ""
+  @State var contactDetails = ""
+  
+  private var borderShape: some InsettableShape {
+    RoundedRectangle(cornerRadius: 4, style: .continuous)
+  }
+  
+  private var textEditor: some View {
+    TextEditor(text: $additionalDetails)
+      .modifier(BaseBraveTextEditorStyleModifier(
+        placeholder: Strings.Shields.reportBrokenAdditionalDetails,
+        isPlaceholderVisible: additionalDetails.isEmpty
+      ))
+  }
+  
+  var body: some View {
+    ScrollView {
+      VStack(alignment: .leading, spacing: 16) {
+        Text(Strings.Shields.reportABrokenSite)
+          .font(.title)
+        Text(Strings.Shields.reportBrokenSiteBody1)
+        Text(url.absoluteString)
+          .foregroundStyle(Color(braveSystemName: .textInteractive))
+        Text(Strings.Shields.reportBrokenSiteBody2)
+          .font(.footnote)
+        
+        textEditor
+        
+        VStack(alignment: .leading, spacing: 4) {
+          Text(Strings.Shields.reportBrokenContactMe).font(.caption)
+          TextField(
+            Strings.Shields.reportBrokenContactMe,
+            text: $contactDetails, prompt: Text(Strings.Shields.reportBrokenContactMeSuggestions)
+          )
+          .textFieldStyle(BraveTextFieldStyle())
+          .autocorrectionDisabled()
+          .textInputAutocapitalization(.never)
+        }
+        
+        HStack {
+          Spacer()
```

Comment:
`.frame(maxWidth: .infinity, alignment: .trailing)` on the `HStack` instead of a spacer

Reply:
The buttons are now on the header.